### ISSUE TITLE
[BUG] fix Python 3 default install dir so special PYTHONPATH is not needed

### ISSUE THREAD

**Issue by siliconja** [coreemu/core#290]
**Describe the bug**
When building from source, you can no longer use 'sudo core-daemon', and have to prepend the command with the appropriate "PYTHONPATH=" setting.

**To Reproduce**
Under Ubuntu 18.04, using `PYTHON=python3 ./configure`, CORE is installed to `/usr/local/lib/python3/dist-packages`, whereas `/usr/local/lib/python3.6/dist-packages` is in the default path.

**Expected behavior**
We should be able to ./bootstrap.sh && configure && make && sudo make install with the minimum number of variables and switches. Default config should be as easy as possible.

**Screenshots**
N/A

**Desktop (please complete the following information):**
 - OS: Ubuntu 18.04
 - CORE Version 5.4.0
 - EMANE Version N/A

**Additional context**
Pull request to address this is coming...

**Comment by bharnden** [comment]
Will look into this, only odd thing at some level is my Ubuntu 18 had */usr/lib/python3* on path, wonder why the difference.

**Comment by siliconja** [comment]
Here is the PR: https://github.com/coreemu/core/pull/292/files

**Comment by bharnden** [comment]
It looks like there is a $(subst find,replace,text) that is an easier way. The other issue that may arise was in trying to build packages for it, was the assumption that python3 and the version specific one would co-exist (I had hoped). Otherwise packaging to a specific version would have its own issues =(.

**Comment by siliconja** [comment]
Do you have a working example of `$(subst ..)`? Because it is not working for me.

I don't follow your comment regarding the "python3" and version-specific (e.g. "python3.6") co-existing.

**Comment by bharnden** [comment]
I pushed a working example, from what I tested to the pull request branch, with an update for ns3 too, I also through in moving the AM_CONDITIONAL out of the if statement to avoid that problem too.

As far as the path issue, it is more like this.

My Ubuntu18 system has python3.6 and the following paths (I don't recall specifically adding python3):
/usr/lib/python3.6
/usr/lib/python3

If another distribution say Ubuntu19 has python3.7 and only a similar path like you have:
/usr/lib/python3.7

If I build the package on Ubuntu18 and it sets the path to /usr/lib/python3.6 that would be an issue. So I was hoping and assuming there would be something constant that could be leveraged beyond having to be version specific, but maybe that is not the case =(.
